### ISSUE TITLE
identity_op false positive

### ISSUE THREAD

**Issue by crumblingstatue** [Manishearth/rust-clippy#189]
```rust
#![feature(plugin)]
#![plugin(clippy)]

fn main() {
    const NANOSECS_PER_SEC: u64 = 1000000000;
    println!("{}", NANOSECS_PER_SEC / 60);
}

```

```
src/main.rs:6:20: 6:41 warning: the operation is ineffective. Consider reducing it to `NANOSECS_PER_SEC`, #[warn(identity_op)] on by default
src/main.rs:6     println!("{}", NANOSECS_PER_SEC / 60);
                                 ^~~~~~~~~~~~~~~~~~~~~

```

This is obviously incorrect.

As far as I can tell, this false positive appeared after #184.

This seems to happen on division/multiplication, but not on addition/substraction.

**Comment by Manishearth** [comment]
It should not be following const bindings now, #184 made sure that it doesn't lookup consts anymore.

cc @llogiq thoughts?

**Comment by llogiq** [comment]
My fault. When converting to const folding, I accidentially removed an important value check.

**Comment by crumblingstatue** [comment]
Thanks for the quick fix!

**Comment by llogiq** [comment]
You're welcome. And sorry for introducing the error in the first place.
